src/mkdocstrings_handlers/python/rendering.py

```diff
@@ -539,11 +539,20 @@ def do_as_attributes_section(
     Returns:
         An attributes docstring section.
     """
+    def _parse_docstring_description(attribute: Attribute) -> str:
+        if attribute.docstring is None:
+            return ""
+        lines = attribute.docstring.value.lstrip().split("\n", 1)
+        if ":" in lines[0]:
+            _, line = lines[0].split(":", 1)
+            lines = [line, *lines[1:]]
+        return lines[0]
```

Comment:
No need to keep track of the `lines` if we just return the first one. Just take the first item of the split (`line= ....split("\n", 1)[0]`) and work on that :slightly_smiling_face: